### ISSUE TITLE
Game idea: Worms

### ISSUE THREAD

**Issue by olizilla** [foundersandcoders/WantToCanvas#2]
Something like http://en.wikipedia.org/wiki/Worms_%28series%29 would work pretty well as an web game. There are a pile of 2D game libraries, and as it's turn based rather than real-time, you can side-step some of the really tricky parts of multiplayer online games.

Props to @liamwooding for the idea, he was thinking of doing it as a Meteor London hack, but it might be more fun if a bunch of us hack on it.

**Comment by harrygfox** [comment]
@olizilla wo YES!

**Comment by liamwooding** [comment]
Yeah let's do it! A simple implementation might look something like the venerable [Scorched Earth](http://en.wikipedia.org/wiki/Scorched_Earth_%28video_game%29).

Basically:
- Draw a squiggly line across the screen to represent the ground
- Place 2 tanks on it
- They take turns guessing the correct angle & power to shoot shells at each other
- Calculate damage based on distance from explosion
- Change the force & direction of the wind each turn
- Repeat until one's health drops to zero

**Comment by FilWisher** [comment]
@liamwooding @olizilla This sounds like an awesome game!

**Comment by graphiclunarkid** [comment]
If you're open to beginners getting involved then count me in for a PR or two :smile:

I've never written an HTML5 application however I've attended a couple of game jams before (see my profile for the results!) @liamwooding's outline looks like a great place to start as it's not too complex.

**Comment by liamwooding** [comment]
@graphiclunarkid all PRs definitely welcome :)
I'll be able to make a start on Friday evening, but for anyone who's interested I think starting with a flat map and getting two players throwing balls at each other would be a good start - at this point we don't really need a proper physics library.
